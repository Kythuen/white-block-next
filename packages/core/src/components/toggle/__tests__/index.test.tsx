import { describe, expect, it } from 'vitest'
import { mount } from '@vue/test-utils'
import Toggle from '../toggle.vue'
// import { useToggle } from '../use'

describe('# Toggle', () => {
  describe('## Base', () => {
    it('### default config', () => {
      const wrapper = mount(Toggle)
      expect(wrapper.classes()).toContain('wb-toggle')
      // expect(wrapper.props()).toEqual({})
      expect(wrapper.element).toMatchSnapshot()
    })
  })

  describe.skip('## Slots', () => {})

  describe.skip('## Events', () => {})

  describe.skip('## Props', () => {})

  describe.skip('## Composables', () => {})
})
