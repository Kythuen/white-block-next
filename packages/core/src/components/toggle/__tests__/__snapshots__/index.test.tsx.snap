// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`# Toggle > ## Base > ### default config 1`] = `
<div
  class="wb-toggle"
>
   Toggle 
  <contentnode />
</div>
`;
